player/src/main/java/net/pubnative/player/VASTPlayer.java

```diff
@@ -634,6 +640,9 @@ public void onSkipClick() {
     public void onOpenClick() {
 
         VASTLog.v(TAG, "onOpenClick");
+        if (mBannerBitmap != null) {
+            mIsBannerSet = true;
+        }
```

Comment:
yes, agree. It's a quick patch. In order to fix it permanently we need to play with setting states of player.